src/color-picker/color-picker.scss

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-color-picker {
+  width: 250px;
+  border-radius: 6px;
+  box-shadow: $iui-elevation-2;
+  display: flex;
+  flex-wrap: wrap;
+  padding-left: 10px;
+  padding-top: 10px;
+  @include themed {  background: rgb(255, 255, 255);}
+
+
+  .iui-color-option {
+    height: 20px;
+    width: 20px;
+    margin-right: 10px;
+    margin-bottom: 10px;
+    cursor: pointer;
+    border-radius: 4px;
+
+    &.iui-light {
+      box-shadow: $iui-elevation-2;
+    }
```

Comment:
Not a fan of this class. Also not a fan of elevation. In this case, other colors have no elevation, giving a very inconsistent look.
![image](https://user-images.githubusercontent.com/9084735/130115662-0c3812ce-3108-4fcd-a557-34b6738399a7.png)

If you apply an inset box-shadow instead, you won't need special handling of light colors, as it will always look good.

Similar idea as this: https://twitter.com/steveschoger/status/1064541476615593984